docs/content/contribute/develop-in-cluster.md

```diff
@@ -135,6 +138,27 @@ oc debug --namespace hypershift deployments/operator --image $(hack/publish-ocp.
 Your latest code should be deployed and logs should soon begin streaming. Just
 press `ctrl-c` to terminate and delete the pod.
 
+## Build and Launch a custom `hypershift-operator` image using a public registry
```

Comment:
Is this the same content already available on this [page](https://hypershift-docs.netlify.app/contribute/custom-images)? Maybe some of this content would be better added there?

Maybe a link could be added here on this page as a reference to that page instead.

Reply:
This page is not accessible with knowing the link. I added a link to it.